contracts/ERC721A.sol

```diff
@@ -693,6 +697,41 @@ contract ERC721A is IERC721A {
         }
     }
 
+    /**
+     * @dev Gets the next extra data for the packed ownership data.
+     */
+    function _nextExtraData(
+        address from,
+        address to,
+        uint256 prevOwnershipPacked
+    ) internal view virtual returns (uint256) {
+        uint24 previousExtraData;
+        assembly {
+            previousExtraData := shr(BITPOS_EXTRA_DATA, prevOwnershipPacked)
+        }
+        return uint256(_extraData(from, to, previousExtraData)) << BITPOS_EXTRA_DATA;
```

Comment:
Is it more expensive to return a uint24 here? In terms of readability, it's more clear what the value here should be if we return uint24.

Reply:
This private function is for returning the shifted extra data.